doc/platforms.md

```diff
@@ -1,16 +1,20 @@
 # Supported Platforms
 
-Flame runs on top of Flutter, so its supported platforms depend on how mature/stable support for each platform is by Flutter.
+Flame runs on top of Flutter, so its supported platforms depend on how mature/stable support for
+each platform is by Flutter.
 
 At the moment, Flame supports both mobile and web.
 
 ## Flutter channels
 
-Flame keeps it support on the stable and beta channel, dev and master channel should work, but we don't support them. This mean that issues happening outside stable or beta channel, are not a priority.
+Flame keeps it support on the stable channel. The dev, beta and master channel should work, but we
+don't support them. This means that issues happening outside the stable channel are not a priority.
 
 ## Flame web
 
-To use Flame on web you need to make sure your game is using the CanvasKit/[Skia](https://skia.org/) renderer, this will increase performance on the web as it will use the `canvas` element instead of seperate HTML elements. 
+To use Flame on web you need to make sure your game is using the CanvasKit/[Skia](https://skia.org/)
+renderer, this will increase performance on the web as it will use the `canvas` element instead of
+seperate HTML elements. 
```

Comment:
```suggestion
To use Flame on web you need to make sure your game is using the CanvasKit/[Skia](https://skia.org/)
renderer. This will increase performance on the web, as it will use the `canvas` element instead of
separate DOM elements. 
```